modules/ding_event/ding_event.module

```diff
@@ -382,6 +413,48 @@ function ding_event_preprocess_views_exposed_form(&$variables) {
         $variables['widgets'][$widget]->widget = $render;
       }
     }
+
+    if ($form['#info']['filter-combine']) {
+      $widget = $form['#info']['filter-combine']['value'];
+      $item = $form[$widget];
+      $render = theme('views_exposed_form_widgets_title', [
+          'element' => [
+            'label' => $variables['widgets']['filter-combine']->label,
+            'field' => $item,
+          ],
+        ]
+      );
+      unset($variables['widgets']['filter-combine']->label);
+      $variables['widgets']['filter-combine']->widget = $render;
+    }
+
+    if ($form['#info']['filter-field_ding_event_date_value'] && $form['#info']['filter-field_ding_event_date_value_1']) {
+      $widget_from = $form['#info']['filter-field_ding_event_date_value']['value'];
+      $widget_to = $form['#info']['filter-field_ding_event_date_value_1']['value'];
+
+      $item_from = $form[$widget_from];
+      $item_to = $form[$widget_to];
+
+      $return = [
+        'from' => [
+          'label' => $variables['widgets']['filter-field_ding_event_date_value']->label,
+          'field' => $item_from,
+        ],
+        'to' => [
+          'label' => $variables['widgets']['filter-field_ding_event_date_value_1']->label,
+          'field' => $item_to,
+        ],
+      ];
+
+      $render = theme('views_exposed_form_widgets_date_inputs', [
+          'element' => $return,
+        ]
+      );
+
+      unset($variables['widgets']['filter-field_ding_event_date_value']->label);
+      $variables['widgets']['filter-field_ding_event_date_value']->widget = $render;
+      unset($variables['widgets']['filter-field_ding_event_date_value_1']);
+    }
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-1115

Reply:
From the issue, I can't read what this change does.

Reply:
It replaces the filters markup with custom templates.

Reply:
Yes, but why and whats the purpose..

Reply:
This one re-arranges the date input filters.
Before:
![image](https://user-images.githubusercontent.com/574498/99979910-51838580-2db0-11eb-9b74-dcd7dbd112c6.png)
After:
![image](https://user-images.githubusercontent.com/574498/99979960-5c3e1a80-2db0-11eb-94a8-e68ddde8f6f0.png)

Reply:
This is not what I see in current ddbasic theme.

Reply:
This is from culture theme.

Reply:
Same as previously - can be implemented in the theme itself to no clutter the module.